### PR TITLE
update harvesting

### PR DESCRIPTION
**Pull request by domakun**

### CODE REVIEW

File: src/views/wallet/wallet-switch/WalletSwitch.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="walletSwitchWrap">
     <div class="walletSwitchHead">
       <p class="tit">{{$t('Wallet_management')}}</p>
-      <p class="back-up pointer" @click="showMnemonicDialog=true">{{$t('backup_mnemonic')}}</p>
+      <p class="back-up pointer" @click="switchMnemonicDialog">{{$t('backup_mnemonic')}}</p>
```
Comment (decentraliser) [inline src/views/wallet/wallet-switch/WalletSwitch.vue]: `showMnemonicDialog`

switch means change

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts [hunk 2]
```diff
@@ -134,6 +137,17 @@ export class WalletSwitchTs extends Vue {
         }
     }
 
+    // @WALLETS refactor
+    switchMnemonicDialog() {
```
Comment (decentraliser) [inline src/views/wallet/wallet-switch/WalletSwitchTs.ts]: showMnemonicDialog

File: src/views/wallet/wallet-switch/WalletSwitch.vue (hunk 1 above)
Comment (domakun) [reply src/views/wallet/wallet-switch/WalletSwitch.vue]: fixed

File: src/views/wallet/wallet-switch/WalletSwitchTs.ts (hunk 2 above)
Comment (domakun) [reply src/views/wallet/wallet-switch/WalletSwitchTs.ts]: fixed

**Comment by decentraliser** [comment]
image

cursor:pointer on button

Link remote account

wording:
Proxy setting: Link remote account
Node configuration: not sure about what it is
Benefit request: Sent delegated harvesting request

**Comment by decentraliser** [comment]
image
i18n keys

not sure about the remote node status, it's a bit confusing.
instead of
Remote node status: ON || N/A
maybe
Remote node status: Online || Unreachable
Would be better

**Comment by decentraliser** [comment]
image

I think it is confusing to always have the Back button on the left, and then to have it on the right in the last screen

**Comment by domakun** [comment]
> Send delegated harvesting request



> image
> 
> I think it is confusing to always have the Back button on the left, and then to have it on the right in the last screen

fixed all

**Pull request closed without merging**